examples/tools/bevymark.rs

```diff
@@ -148,13 +148,14 @@ fn mouse_handler(
 
 fn spawn_birds(
     commands: &mut Commands,
-    window: &WindowDescriptor,
+    windows: &Windows,
     counter: &mut BevyCounter,
     spawn_count: u128,
     texture: Handle<Image>,
 ) {
-    let bird_x = (window.width / -2.) + HALF_BIRD_SIZE;
-    let bird_y = (window.height / 2.) - HALF_BIRD_SIZE;
+    let window = windows.get_primary().unwrap();
+    let bird_x = (window.physical_width() as f32 / -2.) + HALF_BIRD_SIZE;
```

Comment:
Why is this physical width? 

That makes the behaviour with a non-1 scale factor is to spawn the birds off-screen, and let them bounce off-screen. That doesn't seem ideal for a benchmark, as it might make the performance seem better (as off-screen sprites are planned to be culled). I think bevymark is meant to be a stress-test for sprite rendering, not culling behaviour.

Reply:
Yeah my bad, it shouldn't use the physical size, I will fix it.

I agree with your concern for the usefulness of resizing, though the main issue I had here was the use of WindowDescriptor since it gave the wrong impression (to me at least) that WindowDescriptor would be updated.